model/riscv_pmp_regs.sail

```diff
@@ -154,15 +162,28 @@ function pmpReadCfgReg(n) = {
 }
 
 /* Helpers to handle locked entries */
-function pmpLocked(cfg: Pmpcfg_ent) -> bool =
+function pmpLockBit(cfg: Pmpcfg_ent) -> bool =
    cfg.L() == 0b1
+   
+function pmpLocked(cfg: Pmpcfg_ent) -> bool =
+   (cfg.L() == 0b1) & (mseccfg.RLB() == 0b0)
```

Comment:
Maybe better to add a bool argument for whether to allow bypass... pmpcfg.L is still the lock bit and the entry is still locked if set, just in certain situations it can be bypassed when mseccfg.RLB is set